### ISSUE TITLE
the element template chooser modal is missing context for screen reader users

### ISSUE THREAD

**Issue by rpkoller** [bpmn-io/element-template-chooser#15]
__Is your feature request related to a problem? Please describe__
If a screenreader users opens the element template chooser there is no clue in the aural interface that the user is in a modal now and what it's purpose is. it is only announced that the active element is a text field. 

https://user-images.githubusercontent.com/1665422/212682321-8b18ec67-2d40-4171-a1c8-e30fae2104dd.mov

__Describe the solution you'd like__
one option to solve that might be to change the h3 `choose element template` to a label and associate that label with the text field in the modal. 

__Describe alternatives you've considered__

__Additional context__

**Comment by barmac** [comment]
Thanks for opening this issue. Indeed we should have a label for the search field.
